src/config.c

```diff
@@ -844,6 +844,33 @@ int RSConfig_SetOption(RSConfig *config, RSConfigOptions *options, const char *n
   return rc;
 }
 
+void RSConfig_AddToInfo(RedisModuleInfoCtx *ctx) {
+  RedisModule_InfoAddSection(ctx, "run_time_configs");
```

Comment:
Some of these aren't strictly run time configs.
Maybe a better name would `module configurations`?
@K-Jo WDYT?